launch/launch.go

```diff
@@ -31,6 +115,44 @@ type Metadata struct {
 	Buildpacks []Buildpack `toml:"buildpacks" json:"buildpacks"`
 }
 
+// DecodeLaunchMetadataTOML reads a launch.toml file
+func DecodeLaunchMetadataTOML(path string, launchmd *Metadata) error {
```

Comment:
Do we need this custom function anymore actually?

Reply:
Removed